Sources/Public/MonthsLayout.swift

```diff
@@ -48,10 +72,40 @@ extension MonthsLayout: Equatable {
 
   public static func == (lhs: MonthsLayout, rhs: MonthsLayout) -> Bool {
     switch (lhs, rhs)  {
+    case (.vertical(let lhsOptions), .vertical(let rhsOptions)): return lhsOptions == rhsOptions
     case (.horizontal, .horizontal): return true
-    case (.vertical(let l), .vertical(let r)): return l == r
     default: return false
     }
   }
 
 }
+
+// MARK: - VerticalMonthsLayoutOptions
+
+/// Layout options for a vertically-scrolling calendar.
+public struct VerticalMonthsLayoutOptions: Equatable {
```

Comment:
Thoughts on using an option set here instead? Might be a little more straightforward to use in the API.

Reply:
My only hesitation is that there might be non-boolean options that we'd want to add in the future